gemfiles/rails_edge.gemfile

```diff
@@ -1,9 +0,0 @@
-source 'https://rubygems.org'
-
-gem 'bootsnap' # required to run `rake test` in Rails 6.0
-gem 'mocha', '= 1.3' # newer versions blow up
-gem 'rails', git: 'https://github.com/rails/rails.git'
-gem 'rdoc'
-gem 'sqlite3', '~> 1.4'
-
-gemspec path: '../'
```

Comment:
Why not keeping Rails Edge? 7.0 is just around the corner. I have an app running it already.
Maybe just allow-failures on edge